lib/Sema/CalleeCandidateInfo.cpp

```diff
@@ -880,6 +880,26 @@ CalleeCandidateInfo::CalleeCandidateInfo(Type baseType,
     declName = candidates[0].getDecl()->getBaseName().userFacingName();
 }
 
+
+CalleeCandidateInfo &CalleeCandidateInfo::
+operator=(const CalleeCandidateInfo &CCI) {
+  if (this != &CCI) {
+    // If the reference member (i.e., CS) is identical, just copy the rest
```

Comment:
I think `the rest` could be changed to `remaining` here.

Reply:
Fixed.